vllm/v1/engine/__init__.py

```diff
@@ -82,7 +82,12 @@ class EngineCoreOutputs(
     # e.g. columnwise layout
 
     # [num_reqs]
-    outputs: List[EngineCoreOutput]
+    request_ids: List[str]
+    new_token_id_offsets: List[int]
+    new_token_ids: List[int]
+    finished: List[bool]
+    finish_reason: Dict[str, FinishReason]  # Union[List, Dict]?
+    stop_reason: List[Union[int, str, None]]
```

Comment:
Actually stop_reason is only applicable with a finish_reason, so these can just be
```suggestion
    finish_reasons: Dict[str, Tuple[FinishReason, Union[int, str, None]]]
```

Reply:
afaict, the stop reason isn't used anywhere anymore so I got rid of it.  Are there plans to use it in the future?